.kitchen.yml

```diff
@@ -8,8 +8,8 @@ provisioner:
 
 platforms:
   - name: ubuntu-12.04
-  - name: centos-6.6
-  - name: centos-7.1
+  - name: centos-6.8
+  - name: centos-7.3
```

Comment:
@knightsamar please consult with @fahlmant on what I mean when this needs to be fixed.